consensus-types/epbs/indexed_payload_attestation_test.go

```diff
@@ -0,0 +1,53 @@
+package epbs
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestGetAttestingIndices(t *testing.T) {
+	attestingIndices := []primitives.ValidatorIndex{1, 2, 3}
+	
+	nilAttestingIndices := []primitives.ValidatorIndex(nil)
+	pa := &IndexedPayloadAttestation{
+		AttestingIndices: attestingIndices,
+	}
+	got := pa.GetAttestingIndices()
+	require.DeepEqual(t, attestingIndices, got)
+	pa = nil
```

Comment:
```suggestion
	pa = &IndexedPayloadAttestation{
		AttestingIndices: nilAttestingIndices,
	}
```

Reply:
I think this is what you need to do. You can also declare an array of structs each of which corresponds to a test case. For example, you can refer to [this](https://github.com/prysmaticlabs/prysm/blob/develop/api/client/event/utils_test.go#L12).